### ISSUE TITLE
Updating the pre-compiled build dependencies

### ISSUE THREAD

**Issue by darealshinji** [nightingale-media-player/nightingale-deps#33]
The build dependencies on [BitBucket](https://bitbucket.org/nightingale-media-player/nightingale-deps/downloads) for Linux and Windows are still from 2013. Should we update them?

**Comment by freaktechnik** [comment]
I think @ilikenwf recently tried to update the ones for Mac at least. For Linux the plan is to phase deps out, except for taglib and gstreamer for distros that need them (gst1.0 for 12.04/Travis, and taglib for any debian/slackware build, except if you use Ukikie's patched taglib).

I think the gst-1.0 deps don't build for Windows yet. So that's why there's nothing new for windows either.

**Comment by ilikenwf** [comment]
Yes, the mac deps are up to date. Archlinux now builds the one and only
required dep that isn't in the repos (xulrunner) prior to buildding
Nightingale.

**Comment by darealshinji** [comment]
I've uploaded builds that were created with Launchpad in case you want to test them on different distributions: https://github.com/darealshinji/nightingale-deps/releases/tag/20151012

Copies of libjpeg.so.8 and libpng12.so.0 are in included in separate tarballs, since those libraries come in different versions on some distributions.
